Makefile.in

```diff
@@ -1728,6 +1728,7 @@ echo-list	= for x in $1; do $(ECHO) "$$x"; done
 
 .PHONY: all
 all: $(default_pdf_targets) ;
+	killall -q -SIGHUP mupdf || true
```

Comment:
Thats what I assumed. It is actually too specific, but had no clue where the best place for it is.Just played around with it a bit, but could not find a good solution. Maybe we need something like

#
# REFRESH TARGET
#
.PHONY: refresh
refresh: all
	$(QUIET)for x in $(default_pdf_targets); do \
		[ -e "$$x" ] && $(REFRESH_PDF) $$x & \
	done